src/browser/pages/vscode.ts

```diff
@@ -24,13 +24,13 @@ type NlsConfiguration = {
  * Helper function to create the path to the bundle
  * for getNlsConfiguration.
  */
-export function createBundlePath(_resolvedLanguagePackCoreLocation: string, bundle: string) {
+export function createBundlePath(_resolvedLanguagePackCoreLocation: string | undefined, bundle: string) {
```

Comment:
🧹 updated the signature to match the actual value passed in `_resolvedLanguagePackCoreLocation`'s type is `string | undefined`